scripts/build_llvm.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -e
+# Args: LLVM_REPO LLVM_VERSION
+
+if [[ -z "${LLVM_HOME}" ]]; then
+    echo "Error! Environmental variable for LLVM_HOME is not set!"
+    exit 127
+fi
+
+LLVM_REPO=$1
+LLVM_VERSION=$2
+
+if [[ -z "${LLVM_REPO}" ]] || [[ -z "${LLVM_VERSION}" ]]; then
+    echo "Error! Parameters for LLVM repository are incorrect"
+    exit 127
+fi
+
+# Get the source code
+mkdir -p $LLVM_HOME
+cd $LLVM_HOME
+git clone $LLVM_REPO llvm_src
+mkdir build bin
+cd $LLVM_HOME/llvm_src
+git checkout $LLVM_VERSION
+echo $LLVM_REPO:$LLVM_VERSION > $LLVM_HOME/llvm_rev.txt
+git log --format="%H" -n 1 >> $LLVM_HOME/llvm_rev.txt
+
+# Pre-compilation steps
+cd $LLVM_HOME/build
+cmake -G "Ninja" \
```

Comment:
Are we ok building LLVM with system compiler? I think we need a self-build.

Reply:
I'm still suspicious about building test compiler with "random" system compiler.

Ok for this code review, but I'd like to have self-build. If we have powerful machine, it should not be a big deal.

Reply:
It is not random; it is the default one for Ubuntu 20.04. I expect everything to work fine, and we will update the base image eventually anyway.

It takes about 1 hour on AMD EPYC 7502 32-Core Processor with hyperthreading disabled. There is no need to waste 4% of the time without real benefit.